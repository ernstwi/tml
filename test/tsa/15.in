init foo
bar
foo -> bar
    sync { a }
    guard { x < 5 }
    reset { x }
