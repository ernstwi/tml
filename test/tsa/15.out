{
    "actions": [
        "a"
    ],
    "clocks": [
        "x"
    ],
    "edges": [
        {
            "action": "a",
            "from": "foo",
            "guard": "x<5",
            "reset": [
                "x"
            ],
            "to": "bar"
        }
    ],
    "locations": [
        {
            "id": "bar",
            "initial": false,
            "invariant": null
        },
        {
            "id": "foo",
            "initial": true,
            "invariant": null
        }
    ]
}
